README.md

```
@@ -1,253 +1,307 @@
+- Postman: [Link Here](https://www.postman.com)
 
-Open solution in Visual Studio and build the solution (ctrl + shift + b)
+## Docker Installation
 
-## Service Definitions
+1. Run the following terminal commands at the root directory of the repo to download and build Link Cloud and all necessary dependencies: 
 
-### Account Service
+```
+docker compose build
+docker compose up -d		
```

Markdownlint

Column: 21
Hard tabs

(MD010, no-hard-tabs)